pkg/ddl/ddl_api.go

```diff
@@ -6741,8 +6741,8 @@ var systemTables = map[string]struct{}{
 	"gc_delete_range_done": {},
 }
 
-func isSystemTable(schema, table string) bool {
-	if schema != "mysql" {
+func isUndroppableTable(schema, table string) bool {
+	if schema != mysql.SystemDB {
```

Comment:
Is `sys` schema droppable in TiDB?

Reply:
Yes. The `isUndroppableTable` only protects `mysql.tidb`, `mysql.gc_delete_range` and `mysql.gc_delete_range_done`.